network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog_index_feature.html

```diff
@@ -9,7 +9,7 @@
 {% endcomment %}
 
 <div>
-  {% image blog.specific.get_meta_image width-540 %}
+  {% image blog.specific.get_meta_image width-540 height-303 %}
```

Comment:
```suggestion
  {% image blog.specific.get_meta_image fill-540x303 %}
```

Reply:
A cool thing Wagtail has is "focal" points. On uploaded images you can select a part of the image that's important, and when we crop using `fill` it'll try to crop closer to the focal point instead of randomly cropping at an edge or dead center.